assets/js/base/components/product-list/style.scss

```diff
@@ -222,6 +199,70 @@
 	}
 }
 
+.editor-styles-wrapper .wc-block-grid__products .wc-block-grid__product .wc-block-grid__product-onsale,
+.wc-block-grid__product-onsale {
+	border: 1px solid #43454b;
+	color: #43454b;
+	background: #fff;
+	padding: 0.202em 0.6180469716em;
+	font-size: 0.875rem;
+	text-align: center;
+	text-transform: uppercase;
+	font-weight: 600;
+	display: inline-block;
+	width: auto;
+	border-radius: 3px;
+	z-index: 9;
+	position: relative;
+	margin: $gap-smaller auto;
+
+	&.wc-block-grid__product-onsale--alignleft {
+		float: left;
+		margin: 0 auto 0 0;
+	}
+	&.wc-block-grid__product-onsale--alignright {
+		float: right;
+		margin: 0 0 0 auto;
+	}
+}
+.editor-styles-wrapper .wc-block-grid__products .wc-block-grid__product .wc-block-grid__product-image,
```

Comment:
This one is more difficult to remove because it conflicts with a Storefront style. But I think it might be worth investigating how we can reduce Storefront styles' specificity instead of continuing with this specificity war between both projects.

But probably better to do that in a follow-up.